beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -417,7 +417,13 @@ public SafeFuture<Optional<SyncCommitteeContribution>> createSyncCommitteeContri
   }
 
   @Override
-  public void subscribeToBeaconCommittee(final List<CommitteeSubscriptionRequest> requests) {
+  public SafeFuture<Void> subscribeToBeaconCommittee(
+      final List<CommitteeSubscriptionRequest> requests) {
+    return SafeFuture.fromRunnable(() -> processCommitteeSubscriptionRequests(requests));
+  }
+
+  public void processCommitteeSubscriptionRequests(
```

Comment:
Very suspicious that this wound up being public. I'd have thought it would be an internal implementation detail.

Reply:
oops, fixing